style.css

```diff
@@ -386,6 +709,169 @@ body {
   .footer__social-button {
     width: 55px;
   }
+
+  .second-page {
+    min-height: 900px;
+  }
+
+  .second-page__img {
+    margin-right: 166px;
+  }
+
+  .second__img_purses {
+    width: 95%;
+    padding: 102px 0 0 138px;
+  }
+
+  .second__wrap {
+    margin: 50px 73px 0 0;
+  }
+
+  .second-page__article {
+    width: 370px;
+    padding: 40px 61px 46px 57px;
+    margin-bottom: 38px;
+  }
+
+  .second-page__title {
+    font-size: 41px;
+    letter-spacing: -1px;
+    margin-bottom: 6px;
+  }
+
+  .second-page__subtitle {
+    font-size: 16px;
+    letter-spacing: 1.8px;
+    margin-bottom: 63px;
+  }
+
+  .second-page__description {
+    font: 600 14px/21px 'Montserrat-Regular', sans-serif;
+    letter-spacing: .5px;
+    margin-bottom: 38px;
+  }
+
+  .second-page__color-options {
+    margin-bottom: 45px;
+  }
+
+  .eclipse {
+    width: 30px;
+    height: 30px;
+  }
+
+  .second-page__price {
+    margin-right: 12px;
+    margin-bottom: 57px;
+    font-size: 41px;
+  }
+
+  .second__minus {
+    padding: 7px;
+  }
+
+  .second__quantity__input {
+    padding: 7px;
+    width: 25px;
+  }
+
+  .second__plus {
+    padding: 7px;
+  }
+
+  .second__add-to-cart {
+    font-size: 13px;
+    padding: 4px 41px;
+    width: 253px;
+  }
+
+  .second__more-items {
+    font-size: 13px;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #ebe3d8 388px, #000 388px, #000 461px, #ebe3d8 461px);
+    min-height: 900px;
+  }
+
+  .third-page__img {
+    padding-left: 87px;
+  }
+
+  .third-page__img-clothes {
+    width: 100%;
+    padding-top: 0;
+  } 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)